src/server/api-server.js

```diff
@@ -22,9 +24,11 @@ app.use(bodyParser.json())
 app.use('/graphiql', graphiqlExpress({ endpointURL: '/graphql' }))
 app.use('/graphql', graphqlExpress((req, res) => ({ schema })))
 
-app.listen(config.apiPort, err => {
-  const serverName = chalk.bgYellow.bold(' GraphQL Server ')
-  const url = chalk.yellow(`${config.apiHost}:${config.apiPort}`)
+app.use(routers)
```

Comment:
I think the name `routers` makes it seems very important that I have to immediately click it and see what is inside. Is it better to put the code here instead or just rename it to be something more related to `healthz`?